src/appleseed/renderer/kernel/rendering/ipixelrenderer.h

```diff
@@ -86,6 +86,8 @@ class IPixelRenderer
 
     // Return the maximum number of samples per pixel.
     virtual size_t get_max_samples_per_pixel() const = 0;
+
+    virtual void print_settings() const = 0;
```

Comment:
I would find it more logical to find this method at the beginning of the class.

Reply:
No problem, I followed the alphabetical order

Reply:
Should I shift it for all derived and other classes or just this one ?

Reply:
For all classes, I'd say.